chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler2015Version1.java

```diff
@@ -751,7 +751,7 @@ protected void parseMeasures(PendingCertifiedProductEntity listing, PendingCerti
                             }
                             listing.getMeasures().add(measureEntity);
                         }
-                        addAssociatedCriterionToMeasure(measureEntity, cert);
+                        addAssociatedCriteriaToMeasure(measureEntity, cert);
```

Comment:
Is the `addAssociatedCriterionToMeasure` method still used? If not, it ought to be removed

Also, both of these methods have very similar names, and have the same parameters that get passed to them, but using `criteria` in the new one suggests that we're passing in more than one criterion, which I don't see happening. Is there a way to clarify what the difference is in these two methods?

Reply:
I tried renaming them... maybe it's better? One method just adds the criteria associated with the certificationResult that's passed in. I want to do this (and continue to do this) for any measures that aren't found in the database so we can produce an error message. The other method adds the criteria associated with the certificationResult that's passed in AND any other criteria that have the same number. I do this for any valid measure.

Also - I hope this code is removed someday soon in favor of the normalizers in OCD-3549.